java-logger/src/main/resources/archetype-resources/src/main/java/MyLogger.java

```diff
@@ -0,0 +1,21 @@
+package ${package};
+
+import io.streamzi.cloudevents.CloudEvent;
+import io.streamzi.openshift.dataflow.annotations.CloudEventComponent;
+import io.streamzi.openshift.dataflow.annotations.CloudEventConsumer;
+
+import java.util.logging.Logger;
+
+@CloudEventComponent
+public class MyLogger {
+
+    private static final Logger logger = Logger.getLogger(MyLogger.class.getName());
+
+    @CloudEventConsumer(name = "INPUT_DATA")
+    public void onCloudEvent(final CloudEvent evt) {
+
+        evt.getData().ifPresent(data -> {
+            logger.info("Payload: " + data);
```

Comment:
If we would just have Java9, we could use a much much nicer `ifPresentOrElse` function :crying_cat_face: